Doc/library/pty.rst

```diff
@@ -45,9 +45,15 @@ The :mod:`pty` module defines the following functions:
    process's standard io. This is often used to baffle programs which insist on
    reading from the controlling terminal.
 
-   The functions *master_read* and *stdin_read* should be functions which read from
-   a file descriptor. The defaults try to read 1024 bytes each time they are
-   called.
+   The functions *master_read* and *stdin_read* will be passed a file
+   description, and the body should read from this descriptor and
+   then return a byte string. The default implementation for both
```

Comment:
Was there a reason you changed the first line from Martin's suggestion?
`The functions *master_read* and *stdin_read* are passed a file descriptor which they should read from, and should return a byte string.`  I find his wording more concise, yet conveying the same meaning.

Reply:
I was not quite fully awake when I tried to re-write this so I am unsurprised it was less than coherent :laughing: I changed it to Martin's suggestion since I don't recall what problem I thought I had with it.